example/webpack.dev.config.js

```diff
@@ -0,0 +1,17 @@
+const { createConfig } = require('@edx/frontend-build');
+
+const config = createConfig('webpack-dev');
+const resolvedAlias = {};
+
+// When frontend-build tries to resolve aliases defined in module.config.js file
+// it tries to also resolve their peerDependencies by creating additional aliases for them that
+// point to MFE's node_modules, which we do not have since we use npm workspaces in this repo.
+// This loop will simply step out of `example` app's directory and point to Paragon's node_modules
+// for every alias that frontend-build configured to point to `examples`'s app node_modules.
```

Comment:
Appreciate the comment here for the added context! :) I wonder if eventually frontend-build's `module.config.js` file resolution should support monorepos in the future. Maybe a future task for the FWG.